src/__tests__/paxfulapi_test.ts

```diff
@@ -46,6 +50,40 @@ function mockCredentialsStorageReturnValue() {
     });
 }
 
+async function upload_trade_chat_attachment(image: ReadStream | Buffer) {
+    credentialStorage.getCredentials.mockReturnValueOnce({
+        ...expectedTokenAnswer,
+        expiresAt: new Date()
+    });
+
+    const expectedAnswer = {
+        _data: {
+            id: "<id_here>",
+            success: true
+        },
+        status: "success",
+        timestamp: 1455032576
+    };
+
+    (fetch as unknown as FetchMockSandbox).once({
+        url: /https:\/\/api\.paxful\.com\/paxful\/v1\/trade-chat\/image\/upload/,
+        method: "POST",
+        headers: {
+            "Content-Type": "multipart/form-data"
+        }
+    }, {
+        status: 200,
+        body: JSON.stringify(expectedAnswer)
+    }, {
+        sendAsJson: false
+    });
+
+    const paxfulApi = usePaxful(credentials, credentialStorage);
+    const answer = await paxfulApi.invoke(paxfulTradeChatImageUploadUrl, image);
```

Comment:
trade-chat/image/upload beside a binary also accepts "trade_hash" parameter, so we have endpoint which beside binary also accept some parameters

Reply:
Changed the object.